src/libraries/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -837,7 +837,7 @@ internal void ToString(StringBuilder sb)
             {
                 result += SeparatorLiteral + CookieFields.VersionAttributeName + EqualsLiteral + m_version.ToString(NumberFormatInfo.InvariantInfo);
             }
-            return result == EqualsLiteral ? null : result;
+            return result == EqualsLiteral.ToString() ? null : result;
```

Comment:
As an aside, VS tells me that the method containing this code `internal ToServerString()` does not have any callers. So, it may be dead code at this point. We could remove it unless we have a reason to keep it around.

Reply:
https://github.com/dotnet/runtime/blob/f498a3171b65215ae3176598869233fee8e4da58/src/libraries/System.Net.Primitives/src/System/Net/Cookie.cs#L83-L93

Reply:
Ah, nice TIL. Also answered the question I had about trimming too!